### DISCUSSION TITLE
Vue3 Vue.prototype.$route = route alternative

### DISCUSSION THREAD

**Discussion by patie** [inertiajs/inertia#441] [Help (Vue)] [answered]
Hi,

in docs i see **Vue.prototype**.$route = **route**, but this dont work 1. **Vue.prototype** doesnt exists or its bad way (maybe provide os something?) 2. **route** variable/method doesnt exists?

thank you

**Comment by ajnsn** [comment, marked as answer]
@patie In Vue 3 it's [app.config.globalProperties](https://v3.vuejs.org/api/application-config.html#globalproperties)
